tests/helpers/reporter.py

```diff
@@ -0,0 +1,10 @@
+from kitovu.utils import AbstractReporter
+
+
+class TestReporter(AbstractReporter):
+    def __init__(self, raise_errors=False):
+        self.raise_errors = raise_errors
+
+    def warn(self, message):
+        if self.raise_errors:
+            raise Exception(message)
```

Comment:
Without `raise_errors`, shouldn't we at least print them?

Reply:
Printing is a bit pointless if pytest doesn't show anything. Maybe for later tests it would be useful to cache the messages

Reply:
pytest shows the output for failing tests (and you can launch it with `-s` to make them show for all tests). Those warnings are probably useful information for failing tests.